core/arch/aarch64/decode.c

```diff
@@ -67,7 +67,10 @@ byte *
 decode_eflags_usage(dcontext_t *dcontext, byte *pc, uint *usage,
                     dr_opnd_query_flags_t flags)
 {
-    *usage = 0; /* FIXME i#1569 */
+    /* As long as precise flag information is not provided we conservatively
+     * assume each instructions may read or write all flags.
+    */
+    *usage = EFLAGS_WRITE_ALL | EFLAGS_READ_ALL; /* FIXME i#2263 */
```

Comment:
nit: better to have the FIXME be in a full-line comment up above, probably a prefix on the comment on line 70: as it is it sounds like only some minor part of this line is incorrect, and there's no explanation of what the FIXME covers.